baselines/dev/bootstrap.sh

```diff
@@ -2,6 +2,10 @@
 set -e
 cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
 
+# Clone external datasets.
+git clone https://github.com/TalwalkarLab/leaf.git datasets/leaf 2> /dev/null
```

Comment:
Here I'd specify the commit id to checkout after cloning (probably the current head of main branch).